pkg/transport/generic_transport_controller.go

```diff
@@ -494,6 +679,7 @@ func (c *genericTransportController) createOrUpdateWrappedObject(ctx context.Con
 		if err != nil {
 			return fmt.Errorf("failed to create wrapped object '%s' in destination WEC mailbox namespace '%s' - %w", wrappedObject.GetName(), namespace, err)
 		}
+		klog.FromContext(ctx).V(3).Info("Created wrapped object to ITS", "namespace", namespace, "objectName", wrappedObject.GetName(), "wrappedObject", wrappedObject)
```

Comment:
can we remove this log line? 
I assume this was for debugging purposes locally? 
logging every create/update will flood the log with those messages and will make it harder to see errors.

Reply:
I have increased the V level for these debug logging statements.